doc/api/writing-handlers-in-golisp.md

```diff
@@ -35,7 +35,7 @@ So if you are on Windows, and your game is sending an event with the following J
     {
       "game": "MY_GAME",
       "event": "HEALTH",
-      "data": "{\"value\": 50}"
+      "data": "{\"value\": 50, \"frame\": {\"exact-health\": \"2413\"}}"
```

Comment:
Do we want to keep using an escaped JSON string for examples, or switch over to just using raw JSON objects for them?

Reply:
Good point, I added it the way it already was but I don't think it actually makes sense to have it escaped here